### PR TITLE
DT-431 Migrate gnome hitori to Core22/Gnome42

### PR DESCRIPTION
**Pull request by sergio-costas**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please check only the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**Test Configuration**:

* OS (please include version):
* Any other relevant environment information:

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by sergio-costas** [comment]
@hellsworth Removed 'craftctl set version'

File: snapcraft.yaml [hunk 1]
```diff
@@ -34,15 +38,11 @@ parts:
...
     override-build: |
-      sed -i.bak -e 's|=org.gnome.Hitori$|=${SNAP}/meta/gui/org.gnome.Hitori.svg|g' $SNAPCRAFT_PART_SRC/data/org.gnome.Hitori.desktop.in
-      snapcraftctl build
-      mkdir -p $SNAPCRAFT_PART_INSTALL/meta/gui/
-      cp ../src/data/icons/scalable/org.gnome.Hitori.svg $SNAPCRAFT_PART_INSTALL/meta/gui/
-      cp data/org.gnome.Hitori.desktop $SNAPCRAFT_PART_INSTALL/meta/gui/
+      sed -i.bak -e 's|=org.gnome.Hitori$|=${SNAP}/meta/gui/org.gnome.Hitori.svg|g' $CRAFT_PART_SRC/data/org.gnome.Hitori.desktop.in
+      craftctl default
+      mkdir -p $CRAFT_PART_INSTALL/meta/gui/
+      cp ../src/data/icons/scalable/org.gnome.Hitori.svg $CRAFT_PART_INSTALL/meta/gui/
```
Comment (hellsworth) [inline snapcraft.yaml]: use $CRAFT_PART_SRC instead of ..src/data/blah

**Review by hellsworth — CHANGES_REQUESTED** [review]
Minor change requested. Otherwise, looks great!

File: snapcraft.yaml (hunk 1 above)
Comment (sergio-costas) [reply snapcraft.yaml]: Done.

**Review by hellsworth — APPROVED** [review]
Looks good thanks!

**Pull request merged**